test/integration/src/test/java/io/pravega/test/integration/controller/server/ControllerServiceTest.java

```diff
@@ -223,7 +223,7 @@ private static void getSegmentsForNonExistentStream(Controller controller) throw
             
             System.err.println("SUCCESS: Positions cannot be fetched for non existent stream");
```

Comment:
Replaced with log.info(), since its a low risk change I guess its fine to make this change in this PR itself.